src/CodeGeneration/TemplateRender.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Dealroadshow\Bundle\K8SBundle\CodeGeneration;
+
+use InvalidArgumentException;
+
+class TemplateRender
```

Comment:
This class should be named `TemplateRenderer`, with another `er` at the end.
After renaming, we should change variable name as `$renderer` where this class is used.